lib/tasks/codereview.rake

```diff
@@ -0,0 +1,16 @@
+require 'pronto'
+
+desc 'codereview'
+task :codereview do
+  next unless ENV['PULL_REQUEST_ID']
+  next unless ENV['GITHUB_ACCESS_TOKEN']
+
+  puts "checks passed"
+  puts "PR ID: ", ENV['PULL_REQUEST_ID']
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.